components/Form/MultiStepForm.js

```diff
@@ -9,20 +9,31 @@ import Form from 'components/Form/Form';
 import Alert from 'components/Alert/Alert';
 import styles from './MultiStepForm.css';
 
+function validStep(propValue, key, componentName, location, propFullName) {
```

Comment:
Can you move this into the file for custom prop-types?

Reply:
`prop-utils.js`?